
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:2449887) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_1',
  containerIp: '10.0.3.174',
  mitmPort: 1000,
  autoAccessFixed: 1
}
2022-10-22 21:21:29.917 - [Info] MITM Version: 2
2022-10-22 21:21:29.919 - [Info] Auto Access Enabled: true
2022-10-22 21:21:29.919 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-22 21:21:29.926 - [Info] SSH man-in-the-middle server for 10.0.3.174 listening on 10.0.3.1:1000
2022-10-22 21:25:28.238 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.240 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.240 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.241 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.242 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.242 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.249 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.249 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:28.250 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:29.175 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:29.176 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:29.178 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:29.178 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:31.583 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:31.583 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:33.508 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:25:33.508 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:25:39.449 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:25:39.449 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:25:41.221 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'password'
2022-10-22 21:25:41.222 - [Debug] [Auto Access] Attacker: 34.127.21.64, Threshold: 1, Attempts: 1
2022-10-22 21:25:41.222 - [Debug] [Auto Access] Compromising the honeypot
2022-10-22 21:25:41.222 - [Debug] [Auto Access] Adding the following credentials: 'admin:1234'
2022-10-22 21:25:41.333 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-22 21:25:41.335 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-22 21:25:41.335 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.174
2022-10-22 21:25:41.393 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-22 21:25:42.420 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:42.420 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:42.427 - [Debug] [Connection] Attacker connected: 34.127.21.64 | Client Identification: SSH-2.0-Go
2022-10-22 21:25:44.705 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:25:44.705 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:25:47.052 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'password'
2022-10-22 21:25:47.052 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-22 21:25:47.053 - [Debug] [Auth] Attacker: 34.127.21.64 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 21:25:48.003 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:25:48.003 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:25:48.133 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:25:48.133 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:25:50.513 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:25:50.513 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:25:54.033 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'password'
2022-10-22 21:25:54.034 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2022-10-22 21:25:54.035 - [Debug] [Auth] Attacker: 34.127.21.64 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 21:25:57.459 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'password'
2022-10-22 21:25:57.459 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2022-10-22 21:25:57.460 - [Debug] [Auth] Attacker: 34.127.21.64 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 21:25:57.527 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'password'
2022-10-22 21:25:57.528 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-22 21:25:57.528 - [Debug] [Auth] Attacker: 34.127.21.64 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 21:25:59.840 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'password'
2022-10-22 21:25:59.841 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2022-10-22 21:25:59.841 - [Debug] [Auth] Attacker: 34.127.21.64 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 21:26:02.786 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:26:02.786 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:26:03.146 - [Debug] [Auth] Attacker 34.127.21.64 trying to authenticate with 'none'
2022-10-22 21:26:03.146 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 21:26:04.658 - [Debug] [Connection] Attacker closed the connection
2022-10-22 21:26:04.665 - [Debug] [Connection] Attacker closed the connection
